.drone.yml

```diff
@@ -1,3 +1,3 @@
 image: rancher/dind:v0.1.0
 script:
- - ./scripts/ci
+ - ./build.sh
```

Comment:
Should this point to ./scripts/ci  and ./scripts/ci should call ./scripts/bootstrap and ./build.sh.  Bootstrap should `apt-get install make` and call wrapdocker if available